src/style.scss

```diff
@@ -37,15 +44,21 @@ body {
       input {
         width: 2.5em;
         font-size: 2.7em;
+        @media (max-width: $xl) {
+          font-size: calc(1em + 1.9vw);
+        }
         text-align: center;
```

Comment:
This is the general pattern on fonts. Set a max that was nearly identical to your original and don't go any larger on really large displays, but get smaller on smaller displays.